ui/app/routes/application.js

```diff
@@ -140,7 +141,17 @@ export default class ApplicationRoute extends Route {
   @action
   error(error) {
     if (!(error instanceof AbortError)) {
-      this.controllerFor('application').set('error', error);
+      if (
+        error.errors?.any(
+          (e) =>
+            e.detail === 'ACL token expired' ||
+            e.detail === 'ACL token not found'
+        )
+      ) {
+        this.router.transitionTo('settings.tokens');
```

Comment:
`question`: Should a flash message accompany this transition?

Reply:
Token errors (not found, expired) are shown as part of the [Tokens service](https://github.com/hashicorp/nomad/pull/15073/files#diff-06cbf9ecb213715e26c0786cc7972366d6dd1074a4063d4d785283f49eecd0cfR45-R47) — I've added screenshots and some explanations why I'm using errors in service/model instead of here in the application route in the PR notes: https://github.com/hashicorp/nomad/pull/15073#